editor/animation_track_editor.h

```diff
@@ -803,6 +803,9 @@ class AnimationTrackEditor : public VBoxContainer {
 
 	void _anim_paste_keys(float p_ofs, bool p_ofs_valid, int p_track);
 
+	void _toggle_function_name();
```

Comment:
```suggestion
	void _toggle_function_names();
```
I think it better describes it?